stored_requests/backends/file_fetcher/fetcher.go

```diff
@@ -60,6 +60,9 @@ func (fetcher *eagerFetcher) FetchAccount(ctx context.Context, accountDefaultsJS
 		}}
 	}
 
+	if accountDefaultsJSON == nil {
```

Comment:
There are account default defaults, so not sure if this will ever practically come into play.. but fine with the concept.